src/Desktop.Analyzers/UnitTests/MarkAllNonSerializableFieldsTests.cs

```diff
@@ -34,9 +34,14 @@ public class CA2235WithOnlyPrimitiveFields
                     public string s1;
                     internal string s2;
                     private string s3;
+
                     public int i1;
                     internal int i2;
                     private int i3;
+
+                    public bool b1;
+                    internal bool b2;
+                    private bool b3;
```

Comment:
Added tests for `bool`. One community member had reported that `bool` didn't work.